Added by: neurogeek@gentoo.org
Reported to upstream: 21-Jul-2009
Upstream ticket: http://genshi.edgewall.org/ticket/324

diff -uNr Genshi-0.5.1.orig/genshi/template/eval.py Genshi-0.5.1/genshi/template/eval.py
--- Genshi-0.5.1.orig/genshi/template/eval.py	2009-07-21 23:56:48.000000000 +0000
+++ Genshi-0.5.1/genshi/template/eval.py	2009-07-21 23:56:59.000000000 +0000
@@ -229,8 +229,6 @@
     >>> foo = Undefined('foo')
     >>> bool(foo)
     False
-    >>> list(foo)
-    []
     >>> print foo
     undefined
     
